pypeit/core/framematch.py

```diff
@@ -83,15 +83,18 @@ def check_frame_exptime(exptime, exprng):
         ValueError:
             Raised if the length of `exprng` is not 2.
     """
+    # JFH This prevents the code from crashing if there are Nones in the exposure time array, which is possible
```

Comment:
I changed this fix.  Please make sure it makes sense.  Depending on the input range this current edit means that boolean returned for `None`s was unstable.

Reply:
Thanks, your change looks more elegant. To test it I would need a corrupt file though, which I only have when observing. Looks sufficiently straightforward that it must work.

Reply:
Added a test to make sure it works.